tests/fuzz/libfuzz.c

```diff
@@ -0,0 +1,35 @@
+#include <tests/fuzz/libfuzz.h>
+
+#include <ccan/tal/tal.h>
+#include <common/utils.h>
+#include <string.h>
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
```

Comment:
camel case is not a kosher way to write a function name in C? Is there something i'm missing about these names?
```suggestion
int llvm_fuzzer_test_one_input(const uint8_t *data, size_t size);
```

Reply:
> camel case is not a kosher way to write a function name in C?

Agreed

> Is there something i'm missing about these names?

Yeah: this is `libFuzzer`'s API :-)